modules/contact-form/grunion-contact-form.php

```diff
@@ -2038,13 +2038,39 @@ function process_submission() {
 		 * with this work around. */
 		add_filter( 'wp_insert_post_data', array( $plugin, 'insert_feedback_filter' ), 10, 2 );
 
+		$all_values_string = '';
+		
+		// Avoid warnings when print_r() has been disabled on hosting provider for security reasons
+		// function_exists will return false for functions disabled with the disable_functions ini directive
+		if ( function_exists( 'print_r' ) ) {
+			$all_values_string = apply_filters( 'grunion_print_all_values', @print_r( $all_values, true ), $all_values );
```

Comment:
Could you add a docblock before the new filter? Here is an example:
https://github.com/jimmyandrade/jetpack/blob/fa1c63e9ee9b0a2d21a0855e5f582574888554be/modules/contact-form/grunion-contact-form.php#L526-L535

You can learn more about Hook documentation standards here:
https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters